content/en/tracing/trace_collection/library_config/nodejs.md

```diff
@@ -26,87 +26,211 @@ Tracer settings can be configured with the following environment variables:
 ### Tagging
 
 `DD_ENV`
-: Set an application's environment (for example, `prod`, `pre-prod`, and `stage`). Defaults to the environment configured in the Datadog Agent.
+: **Configuration**: `env`<br>
+**Default**: Defaults to the environment configured in the Datadog Agent<br>
+Set an application's environment (for example, `prod`, `pre-prod`, and `stage`).
 
 `DD_SERVICE`
-: The service name used for this program. Defaults to the name field value in `package.json`.
+: **Configuration**: `service`<br>
+**Default**: Defaults to the `name` field in `package.json`<br>
+The service name used for this program.
 
 `DD_VERSION`
-: The version number of the application. Defaults to the version field value in `package.json`.
+: **Configuration**: `version`<br>
+**Default**: Defaults to the `version` field in `package.json`<br>
+The version number of the application.
 
 `DD_TAGS`
-: Set global tags that are applied to all spans and runtime metrics. When passed as an environment variable, the format is `key:value,key:value`. When setting this programmatically, the format is `tracer.init({ tags: { foo: 'bar' } })`.
+: **Configuration**: `tags`<br>
+**Default**: `{}`<br>
+Set global tags that are applied to all spans and runtime metrics. When passed as an environment variable, the format is `key:value,key:value`. When setting this programmatically, the format is `tracer.init({ tags: { foo: 'bar' } })`.
 
 It is recommended that you use `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` to set `env`, `service`, and `version` for your services. Review the [Unified Service Tagging][1] documentation for recommendations on configuring these environment variables.
 
 ### Instrumentation
 
 `DD_TRACE_ENABLED`
-: **Default**: `true`<br>
-Whether to enable the tracer.
+: **Configuration**: N/A<br>
+**Default**: `true`<br>
+Whether to enable dd-trace. Setting this to `false` will disable all features of the library.
```

Comment:
We should probably have a discussion about if it's confusing to document both `DD_TRACE_ENABLED` and `DD_TRACING_ENABLED`. Can we make a different env var for the global disable to make the distinction clearer and deprecate this one?

Reply:
I agree that its confusing. At first glance it seems to be a mistake on our part for offering both. @Qard @rochdev do either of you know how these two features differ?

Reply:
I don't think we can deprecate `DD_TRACE_ENABLED` as it appears to be a Datadog standard that is used by other libraries. We could rename `DD_TRACING_ENABLED` though.

Reply:
Some APM folks across different teams are discussing the future of these values. We'll likely change wording and rename one in the future.